### PR TITLE
provide UI for opening '.rds' files

### PR DESCRIPTION
**Pull request by kevinushey**
This PR adds a small feature-let for loading `.rds` files when clicked on from the Environment pane:

screen shot 2017-08-28 at 3 42 49 pm

screen shot 2017-08-28 at 3 43 06 pm

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/common/filetypes/RDSDataType.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/*
+ * RDSDataType.java
+ *
+ * Copyright (C) 2009-12 by RStudio, Inc.
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/common/filetypes/RDSDataType.java]: Nit: 2009-17

**Comment by jmcphers** [comment]
There's no need to take this for 1.1 but the risk is exceedingly low so I'm fine with merging it there.

File: src/gwt/src/org/rstudio/studio/client/workbench/views/environment/EnvironmentPresenter.java [hunk 2]
```diff
@@ -548,24 +550,48 @@ void onImportDatasetFromMongo()
    public void onOpenDataFile(OpenDataFileEvent event)
    {
       final String dataFilePath = event.getFile().getPath();
-      globalDisplay_.showYesNoMessage(GlobalDisplay.MSG_QUESTION,
-           "Confirm Load RData",
+      if (dataFilePath.endsWith(".rds"))
```
Comment (jmcphers) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/environment/EnvironmentPresenter.java]: Does this need to be case-insensitive?

**Comment by jmcphers** [comment]
Nice, thanks! Could you add a NEWS entry for this?

**Pull request merged**